config/config.default.json

```diff
@@ -20,7 +20,8 @@
     },
     "UsersConfiguration": {
         "InitialActiveUsers": 4,
-        "MaxActiveUsers": 1000
+        "MaxActiveUsers": 1000,
+        "AvgSessionsPerUser": 1
```

Comment:
Just wanted to note that this change will probably make load-tests fail on new deployments unless we also update:

https://github.com/mattermost/mattermost-load-test-ng/blob/b8b58286e858a5d09cef7cef9736311339083bef/deployment/terraform/agent.go#L32-L35

Reply:
Ahh .. we need to get rid of that. Going to be problematic in future.

Reply:
Yes. We have https://mattermost.atlassian.net/browse/MM-23819 which I'll try to prioritize.